plugins/modules/listen_ports_facts.py

```diff
@@ -113,7 +113,7 @@
           type: str
           sample: "mysqld"
         pid:
-          description: The pid of the listening process.
+          description: The pid of the listening process. If user permissions not allowed, set to C(0).
```

Comment:
I would put this in a second paragraph. Also use `V()` for values.
```suggestion
          description:
            - The pid of the listening process.
            - If user permissions not allowed, set to V(0).
```